tests/acceptance/bootstrap/FeatureContext.php

```diff
@@ -2447,7 +2447,15 @@ public function substituteInLineCodes(
 					"getXRequestIdRegex"
 				],
 				"parameter" => []
-			]
+			],
+			[
+				"code" => "%created_app_token%",
```

Comment:
we should say if it is a pattern or not
```suggestion
				"code" => "%app_token_pattern%",
```

Reply:
maybe `short_app_token_pattern` and `long_app_token_pattern` if we need both